lib/seahorn/CexExeGenerator.cc

```diff
@@ -221,7 +229,7 @@ void CexExeGenerator<Trace>::buildNonDetFunction(const Function *func,
     else
       Args.push_back(ConstantInt::get(Type::getInt32Ty(m_context), 0));
   } else {
-    errs() << "WARNING: Unknown type: " << *RT << "\n";
+    ERR << "WARNING: Unknown type: " << *RT << "\n";
```

Comment:
either `ERR` or `WARN` but then no `WARNING` since the prefix is added automatically